src/OrchardCore/OrchardCore.Roles.Abstractions/Extensions/SystemRoleProviderExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using OrchardCore.Security;
+
+namespace OrchardCore.Roles;
+
+public static class SystemRoleProviderExtensions
+{
+    public static async ValueTask<IRole> GetAdminRoleAsync(this ISystemRoleProvider systemRoleNameProvider)
+        => (await systemRoleNameProvider.GetSystemRolesAsync()).First();
```

Comment:
Don't depend on role position is the collection. If someone swaps implementation, this may break.

Reply:
This is much faster, I can add a comment to avoid change the position

Reply:
It's not good practice to add an extensions of an interface where the extension depends on implementation details of a concrete class. It better in this case to have the method as part of the interface since it is required.